docs/source/_templates/sdk_TP_tutorial.rst

```diff
@@ -441,3 +441,171 @@ about what kinds of transactions it can handle.
 
 
 {% endif %}
+
+
+
+Playing XO With The XO Client
+=============================
+
+You can now use the included XO client to test your new XO transaction
+processor. Sawtooth includes an XO CLI-based client written in Python, which
+can interact with any correctly written XO transaction processor, regardless
+of language.
+
+The steps below show you how to set up and play a game.
+
+
+Start The Necessary Components
+------------------------------
+
+To play tic-tac-toe, you need the following components to be running and
+connected:
+
+#. At least one validator
+#. The config family transaction processor, tp_config
+#. An XO family transaction processor (the one you have written, or one
+   of the included XO processors)
+#. The REST API
+
+If you are using a Docker environment with the Docker Compose file provided in
+:doc:`/app_developers_guide/docker`, then the *tp_xo_python* transaction
+processor is automatically started.
+
+For more information on configuring and runing Sawtooth components, see
+:doc:`/app_developers_guide/getting_started`.
+
+
+Create Players
+--------------
+
+Create your first player:
+
+.. code-block:: console
+
+    $ xo init --username jack
+
+The command produces output similar to:
+
+.. code-block:: console
+
+    set username: jack
+    set url: http://127.0.0.1:8080
+    writing file: /home/ubuntu/.sawtooth/keys/jack.priv
+    writing file: /home/ubuntu/.sawtooth/keys/jack.addr
+
+.. warning::
+
+    The file paths to your keys will reflect your environment.
+
+
+Create player number two:
+
+.. code-block:: console
+
+    $ xo init --username jill
+
+The command produces output similar to:
+
+.. code-block:: console
+
+    set username: jill
+    set url: http://127.0.0.1:8080
+    writing file: /home/ubuntu/.sawtooth/keys/jill.priv
+    writing file: /home/ubuntu/.sawtooth/keys/jill.addr
+
+
+Create A Game
+-------------
+
+Create a game with the following command:
+
+.. code-block:: console
+
+    $ xo create game1
+
+To see list of the created games, enter the following command:
+
+.. code-block:: console
+
+    $ xo list
+
+The command outputs a list of the games that have been created:
+
+.. code-block:: console
+
+    GAME            PLAYER 1        PLAYER 2        BOARD     STATE
+    game1                                           --------- P1-NEXT
+
+
+Take A Space As Player One
+--------------------------
+
+Start playing by taking a space as the first player:
+
+.. code-block:: console
+
+    $ xo take game1 4
+
+.. note::
+
+    The board spaces are numbered from one to nine. The upper-left corner is
+    number one, and the lower right corner is number nine.
+
+
+Take A Space As Player Two
+--------------------------
+
+Take a space on the board as player two:
```

Comment:
I was wrong. You do have to switch players manually. The game lets you play against yourself! ;)